.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+# Describe the bug
+A clear and concise description of what the bug is.
+
+# To reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+# Expected behavior
+A clear and concise description of what you expected to happen.
+
+# Screenshots / Videos
+If applicable, add screenshots or videos to help explain your problem.
+
+# Environment (please complete the following information
```

Comment:
also, if possible, output of `pip freeze` or similar to describe all installed packages and their versions

Reply:
Added, good call out